scripts/deploy_release.sh

```diff
@@ -0,0 +1,32 @@
+#/bin/bash -e
+# Copyright 2023 The ChromiumOS Authors
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+TMPDIR=`mktemp -d`
+if ! [ -z "$(git status --porcelain)" ]; then
+	echo "Uncommited changes found. Please commit the change first."
+	exit 1
+fi
+if ! git status | grep 'up to date' ; then
+	echo "Remote branch is not up to date. Please push the commits first."
+	exit 1
+fi
+cargo test --target-dir ${TMPDIR}
+cargo build --release --target-dir ${TMPDIR}
+VERSION=`${TMPDIR}/release/lium version | cut -d ' ' -f 2`
+
+if gh release list | grep ${VERSION} ; then
+	# Release with the version found. Delete it first.
+	gh release delete -y ${VERSION}
+fi
+
+# Create a new release
+read -r -d '' RELEASE_NOTE <<EOF
```

Comment:
I see that a user can figure out the version from the curl command line, but still it might be a good idea to add another line stating the version explicitly in the first line. e.g. LIUM version v0.1.1.

Although I guess it's not strictly necessary at this point. Feel free to close this comment by yourself.

Reply:
I think we don't need that for now and it can be easily noticed by seeing the command line. it would be appreciated if you allow me to proceed with this.